### PR TITLE
no recording for mobile screen

### PR DESCRIPTION
**Pull request by mritunjayz**

### CODE REVIEW

**Comment by mritunjayz** [comment]
solves #2178

File: web/locales/en/messages.ftl [hunk 1]
```diff
@@ -324,6 +324,7 @@ off = Off
 on = On
 add-avatar-title = Add an avatar to your profile
 add-avatar-clip = Add an avatar to your profile and say “hello!”
+avatar-clip-title = An avatar to your profile!
```
Comment (Gregoor) [inline web/locales/en/messages.ftl]: this english looks incomplete

File: web/src/components/pages/profile/avatar-setup/avatar-setup.css [hunk 2]
```diff
@@ -26,9 +26,23 @@
         left: 15%;
         right: 15%;
         padding-top: 10px;
-        @media only screen and (max-width: 375px) {
+
+        @media only screen and (max-width: 415px) {
             left: 6%;
             right: 6%;
+            display: none;
```
Comment (Gregoor) [inline web/src/components/pages/profile/avatar-setup/avatar-setup.css]: I don't think we want to actually hide the recording on mobile in general. Rather only on browsers that don't support recording

File: web/locales/en/messages.ftl (hunk 1 above)
Comment (mritunjayz) [reply web/locales/en/messages.ftl]: I just wanna ask this after opening a PR ! that what should I show as the title if no recording option is available !

Comment (mritunjayz) [reply web/locales/en/messages.ftl]: No, it is not from copy doc

**Comment by mritunjayz** [comment]
I think for now we can go for this until the design is not ready for it

**Pull request merged**